dalog/convert.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from "fs";
+import { execSync } from "child_process";
+import readXlsxFile from "read-excel-file/node";
+import type { FeatureCollection } from "geojson";
+
+async function main() {
+  let gj: FeatureCollection = {
+    type: "FeatureCollection",
+    features: [],
+  };
+
+  let file = fs.readFileSync(process.argv[2]);
+
+  // The automatic column detection doesn't work, because there's an empty row and column at the start
+  let rows = await readXlsxFile(file, {
+    sheet: "Issues",
+  });
+  let keys = rows[1].slice(1);
+  for (let row of rows.slice(2)) {
+    let obj = Object.fromEntries(keys.map((key, i) => [key, row[i + 1]]));
+    if (obj["Issue ID"] == null) {
+      continue;
+    }
+
+    let ll = obj["Latitude & Longitude"];
```

Comment:
This might've been fixed in the live copy of the data; if you don't see a warning when running, then please remove this bit